app/src/ereferrals/java/org/eyeseetea/malariacare/domain/usecase/CompletionSurveyUseCase.java

```diff
@@ -1,91 +1,7 @@
 package org.eyeseetea.malariacare.domain.usecase;
 
-import android.support.annotation.Nullable;
-
-import com.google.common.base.Function;
-import com.google.common.collect.Maps;
-
-import org.eyeseetea.malariacare.data.database.model.Question;
-import org.eyeseetea.malariacare.data.database.model.Value;
-import org.eyeseetea.malariacare.data.database.utils.Session;
-import org.eyeseetea.malariacare.domain.entity.Survey;
-import org.eyeseetea.malariacare.utils.Constants;
-
-import java.util.List;
-import java.util.Map;
-
 public class CompletionSurveyUseCase extends ACompletionSurveyUseCase {
-    Function<Value, Question> valuesToQuestions = new Function<Value, Question>() {
-        @Nullable
-        @Override
-        public Question apply(Value value) {
-            return value.getQuestion();
-        }
-    };
-    private Survey mSurvey;
-
     @Override
     public void execute(long idSurvey) {
-        Survey survey = getSurveyWithStatusAndAnsweredRatio(idSurvey);
-        updateRDTStockQuestion(survey);
-    }
-
-    private Survey getSurveyWithStatusAndAnsweredRatio(long idSurvey) {
-        org.eyeseetea.malariacare.data.database.model.Survey surveyDB =
-                org.eyeseetea.malariacare.data
-                        .database.model.Survey.findById(idSurvey);
-        Survey survey = new Survey(idSurvey);
-        survey.setSurveyAnsweredRatio(surveyDB.reloadSurveyAnsweredRatio());
-        surveyDB.updateSurveyStatus();
-        survey.setStatus(surveyDB.getStatus());
```

Comment:
Are you sure that removing this part won't cause your survey to never pass to complete state? Probably the name of the function is badly chosen because a function named getSomething() should never do a permanent change in the DB, but anyway I think we need to keep it and rename the function. Unless you tell me that you moved elsewhere...but if that's the case I don't see that change in this PR.